src/routes/Router.tsx

```diff
@@ -32,14 +38,32 @@ const Router = () => {
         <Route path="/legal/termsAndConditions" element={<Legal document="termsAndConditions" />} />
         <Route path="/legal/privacyPolicy" element={<Legal document="privacyPolicy" />} />
         <Route path="/legal/cookiePolicy" element={<Legal document="cookiePolicy" />} />
+        {renderLegacyRoute(legacyCreateBoard)}
         <Route
-          path="/new"
+          path="/boards"
           element={
             <RequireAuthentication>
-              <NewBoard />
+              <Boards />
             </RequireAuthentication>
           }
-        />
+        >
+          <Route index element={<Navigate to="templates" />} />
+          <Route path="templates" element={<Templates />}>
+            {/* TODO extract settings routes to avoid repetition */}
+            <Route path="settings" element={<SettingsDialog enabledMenuItems={{appearance: true, feedback: feedbackEnabled, profile: true}} />}>
+              <Route path="appearance" element={<Appearance />} />
+              <Route path="feedback" element={<Feedback />} />
+              <Route path="profile" element={<ProfileSettings />} />
```

Comment:
Both the appearance and profile menu item have the same background-color on hover

Reply:
yeah it's both `poker-purple, which is also like it would be with the full settings range. but yeah makes sense to change one of those, maybe to `value-violet`. need to look into how I can make the colors overwritable

Reply:
as mentioned above changing one of the constant colors would've been the easy solution, but since it's a good idea to allow for dynamic changes I made this PR allowing to overwrite the settings, and consequently the color: https://github.com/inovex/scrumlr.io/pull/4854